docs/proposals/platform-extension-implementation-extension-operators.md

```diff
@@ -0,0 +1,96 @@
+# Platform Extension Implementation: Extension Operators
+
+**Author**: Moritz Bracht
+
+**Status**: Draft proposal.
+
+## Goals
+
+This proposal explores an operator based approach for KKP extensions.
+
+## Motivation and Background
+
+An implementation of platform-extension based on Helm charts provides flexible yet simplistic way to
+manage the lifecycle of platform-extensions.
+
+In case of more complex extensions that can not simply be installed with Helm ([like
```

Comment:
KubeVirt could be easily installed with helm chart, but deleting KubeVirt is a problem, there is a need for some customization.
> Note: The apiservice and the webhookconfigurations need to be deleted manually due to a bug.